test/specs/lit.ts

```diff
@@ -79,6 +88,17 @@ test('lit-css', (t) => {
           color yellow
           background url("data:image/gif;base64,R0lGO")
         }
+        .cls {
+          &:before {
+            content: '';
+            position: absolute;
+            border-width: 0;
+          }
+          &:after {
+            content: '';
+            display: block;
+          }
+        }
         // .cls:after {
```

Comment:
If you want to test this issue https://github.com/houd1ni/fela-vue/issues/17
then better to uncomment these lines, because the new tests above worked perfectly well before the issue.

Reply:
Why the commented should be involved? There wasn't issue with such option of typing.
Remove all parts that I added and just uncomment your part: it means we don't need to consider the first comment?

Reply:
Where I've said they should be removed? It's ok to test both cases, I've only been thinking, you wanted to test you bug-case after minification? Then leave both cases concatinated and not along 🌵

Reply:
okay, let's keep both cases

Reply:
okay, let's keep both cases

Reply:
Thank you!